### PR TITLE
Russian translation of TOML 0.5.0

### PR DESCRIPTION
**Pull request by Suhoy95**
Signed-off-by: Ilya Sukhoplyuev suhoy@gramend.ru
(Gramend R&D and Innopolis University, SNE master)

### CODE REVIEW

**Comment by lmna** [comment]
Overall quality of this translation is very poor. It has way too many defects to enumerate them here.

**Comment by Suhoy95** [comment]
Hello, thanks for fast reply

Well, It is really possible, especially because it is strange to expect of brilliant quality of this one day crusade to the TOML land. I just decided that making translation is better than just reading for my education. The other benefits for community is secondary. 

Even more, I am not satisfied with quality of english TOML version (#700). I am reading RFCs, IEEE and Microsoft standards, and TOML "specification" is a little bit far away from it. I guess this adventure should be main purpose to introduce TOML 1.0.0 . (For beta-version TOML spec is acceptable)

> It has way too many defects to enumerate them here.

It is really strange sentence for me. I put part of my soul into that and I became very attached to the translation, maybe too much than It should be. But wait, why you write it? Why do you spend your precious time for writing this? You could just go and do your, I guess, more important things. GitHub has line comments exactly for this purpose, also I put " Allow edits from maintainers." flag, so if you are, you can fix all the defects and merge even not to intervent me at all.

File: versions/ru/toml-v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,906 @@
...
+
+Автор: Том Престон-Вернер (Tom Preston-Werner).
+
+Версия TOML 0.5.0 должна рассматриваться как предельно стабильная. Цель версии
+1.0.0  быть обратно совместимой с версией 0.5.0 (на сколько это возможно,
+с сохранением читаемости для человека). Авторам TOML имплементаций строго
+рекомендуется реализовать совместимость с версией 0.5.0, что позволит
+без особых затрат выполнить обновление к версии 1.0.0, когда это случиться.
+
+Переводчик
```
Comment (skatkov) [inline versions/ru/toml-v0.5.0.md]: I think it's irrelevant who the translator is and who he works for. Can we please drop this from translation?

**Comment by skatkov** [comment]
It's a tough material to digest. It would be nice if you can take another stab at it.

A lot of things throw me off while reading:
- the translation from English in some case is too direct, Russians don't really speak like that. It requires a couple of "takes" to understand.
- "notes from author" seem completely inappropriate
- English terms in brackets throw me back to "English mode" and reading goes really slow...

**Comment by pradyunsg** [comment]
Closing since the released versions of the TOML specification, as well as their translations, are hosted on https://github.com/toml-lang/toml.io now.

**Pull request closed without merging**